object_types: ['door', 'cone']
object_file_locations: ['/files/doors.csv', '/files/cones.csv']
object_publisher_topic: ['detected_doors', 'detected_cones']
